src/libraries/Microsoft.Extensions.Http/src/Logging/LogHelper.cs

```diff
@@ -57,6 +67,11 @@ private static class EventIds
             EventIds.PipelineEnd,
             "End processing HTTP request after {ElapsedMilliseconds}ms - {StatusCode}");
 
+        private static readonly Action<ILogger, double, string?, Exception?> _requestPipelineFailed = LoggerMessage.Define<double, string?>(
+            LogLevel.Error,
+            EventIds.PipelineFailed,
+            "Request failed after {ElapsedMilliseconds}ms, Error: {ErrorMessage}");
```

Comment:
Same here with regards to the unified wording and the ErrorMessage parameter: `"Error while processing HTTP request after {ElapsedMilliseconds}ms"`

Reply:
@CarnaViire done

Reply:
The message is not updated -- can you pls check (maybe the changes were not committed/pushed?)